index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Awesome Books</title>
+    <link rel="stylesheet" href="index.css" />
+  </head>
+  <body>
+    <!-- Header and navigation section -->
+    <header class="header">
+      <nav class="nav-container">
```

Comment:
- [ ] Hi, Your Navbar is not working as mentioned in the preview review, We need to check if it is possible to add and remove books from the list.
- [ ] I would advise you not to forget to display the current date below the navigation bar. It is part of the requirements
![Capture](https://user-images.githubusercontent.com/63972670/220879204-49f14556-281e-46d1-973c-5056472013f5.PNG)

Reply:
Hi @OLIPLICHE 
 The navbar is working with me if I use a live server
![Add-work](https://user-images.githubusercontent.com/51516029/220890321-9891955d-c5b9-4b94-b46f-06baf5e684c2.png)
![nav-bar-work](https://user-images.githubusercontent.com/51516029/220890398-a6a58af4-2a39-41e9-ab6e-95835f63a1a3.png)
but if I try to use GitHub pages it can't work. I need your help.